### PR TITLE
2022 MDs Page

### PR DESCRIPTION
**Pull request by fsoubelet**
Adds MD in the 2022 measurements page with info on the 3 MDs done last week in block 2.
Need to add the ones from block 1 but I wasn't there, will have a look later.

[Download build](https://github.com/pylhc/pylhc.github.io/suites/9321482960/artifacts/437932368)

### CODE REVIEW

File: docs/resources/logbook/2022_lhc.md [hunk 1]
```diff
@@ -1,58 +1,63 @@
...
+# Timeline of LHC Run 3 Measurements in 2022
 
-This page provides quick access links to logbook entries of important OMC shifts or measurements done during the LHC 2022 commissioning.
-Some `Shift Purpose` entries (underlined) offer a tooltip when hovered, with a quick description of the important information from that entry.
-The table below can be sorted by clicking next to the column headers.
+This page provides quick access links to logbook entries of important OMC shifts or measurements done during the LHC 2022 commissioning and MDs.
+Some entries (underlined) offer a tooltip when hovered, with a quick description of the important information from that entry.
+The tables below can be sorted by clicking next to the column headers.
+
+## LHC 2022 Commissioning
```
Comment (JoschD) [inline docs/resources/logbook/2022_lhc.md:7]: not sure if this is still needed

File: docs/resources/logbook/2022_lhc.md [hunk 2]
```diff
@@ -1,58 +1,63 @@
...
+| 2022-11-06 |    MD 8523    | Amplitude Detuning in the Squeeze and Skew-Octupole Measurements at 30cm |                                                         [Shift Plan][md_8523_plan]{target=\_blank .cern_login} /  [Summary][md_8523_summary]{target=\_blank .cern_login}                                                         |
+| 2022-11-07 |    MD 7203    |                           Rematched IR7 Optics                           |                                                         [Shift Plan][md_7203_plan]{target=\_blank .cern_login} / [Summary][md_7203_summary]{target=\_blank .cern_login}                                                          |
+| 2022-11-07 |    MD 7003    |                          2023 LHC Configuration                          |                                                         [Shift Plan][md_7003_plan]{target=\_blank .cern_login} / [Summary][md_7003_summary]{target=\_blank .cern_login}                                                          |
 
 
-<!-- All the links below -->
+<!-- All the commissioning links below -->
```
Comment (JoschD) [inline docs/resources/logbook/2022_lhc.md:60]: also it's not all links again

**Review by JoschD — APPROVED** [review]

File: docs/resources/logbook/2022_lhc.md (hunk 1 above)
Comment (fsoubelet) [reply docs/resources/logbook/2022_lhc.md]: fixed

File: docs/resources/logbook/2022_lhc.md (hunk 2 above)
Comment (fsoubelet) [reply docs/resources/logbook/2022_lhc.md]: fixed

**Review by JoschD — APPROVED** [review]

**Pull request merged**